modules/features/player/src/main/java/au/com/shiftyjelly/pocketcasts/player/view/PlayerContainerFragment.kt

```diff
@@ -120,17 +122,20 @@ class PlayerContainerFragment : BaseFragment(), HasBackstack {
 
             override fun onPageSelected(position: Int) {
                 super.onPageSelected(position)
-                when (position) {
-                    0 -> {
+                when (adapter.pageTitle(position)) {
+                    LR.string.player_tab_playing -> {
```

Comment:
This seems to link back to `VR.string.player_tab_playing` and has different values, either `Now Playing` or `Playing` depending on the screen width. 

I wonder if it would be more explicit with something like:

```
when {
    adapter.isPlayerTab(position) -> {
```

```
fun isPlayerTab(position: Int): Boolean {
    return sections[position] is Section.Player
}
```

Just an idea, feel free to ignore it.

Reply:
Suggestion applied to make it more explicit in 2342c47